docs/release-notes/2.4/2.4.29.md

```diff
@@ -0,0 +1,27 @@
+# Percona XtraBackup 2.4.29 (2023-12-)
+
+!!! important
+
+    Percona XtraBackup 2.4.29 is the final release of the Percona XtraBackup 2.4 series.
+
+    We recommend that you either [upgrade to MySQL 8.0 or stay on 5.7; we'll support you](https://www.percona.com/navigating-mysql-5-7-end-of-life).
```

Comment:
I don't know if we need the reference to 8.0/5.7. Let's remove it.

Reply:
fixed